recipe/meta.yaml

```
@@ -36,36 +34,34 @@ requirements:
     - geotiff
     - giflib   # [not win]
     - hdf4
-    - hdf5
-    - icu
+    - hdf5 1.12.1
     - jpeg
     - json-c   # [not win]
     - kealib
     - lerc
-    - libcurl
+    - libcurl 7.88.1
     - libdeflate 1.17
     - libiconv
     - libkml
-    - libnetcdf
+    - libnetcdf 4.8.1
     - libpng
-    - libpq
+    - libpq 12.15
     - libspatialite
     - libtiff
     - libuuid  # [linux]
     - libwebp-base
     - libxml2 2.10
```

```suggestion
    - libxml2 {{ libxml2 }}
```
As we already have 2.10 in cbc.yaml